src/test/resources/com/puppycrawl/tools/checkstyle/checks/orderedproperties/ExpectedOrderedProperties1OrderKey.properties

```diff
@@ -0,0 +1,3 @@
+key1=value2
```

Comment:
If these files are used somewhere please point me to them. We should only need `Input` files here.

Reply:
I removed this file.
Done.